CI/travis/lib.sh

```diff
@@ -12,13 +12,26 @@ fi
 
 export PYTHON3=python3
 
-get_codename() {
-	lsb_release -a 2>/dev/null | grep Codename | cut -f2
+version_gt() { test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" != "$1"; }
+version_le() { test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" == "$1"; }
+version_lt() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }
+version_ge() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" == "$1"; }
+
+get_dist_id() {
+	lsb_release -i -s
+}
+
+get_version() {
+	lsb_release -r -s
+}
+
+version_gt() {
```

Comment:
Isn't this a redefinition of the version_gt() above?

Reply:
yep; thanks for catching;
i think i tried too many things at once and got that from 2 different sources and forgot to remove one